app/admin/projects.rb

```diff
@@ -86,6 +88,10 @@
       row 'Approved?' do ad.is_approved end
       row :created_at
       row :updated_at
+      row :twitter do |proj|
+        link_to proj.twitter, 'http://twitter.com/' + proj.twitter, 
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.